tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_activation.py

```diff
@@ -74,3 +74,41 @@ def test_keras_activation_float32(self, params, ie_device, precision, ir_version
         self._test(*self.create_keras_activation_net(**params, ir_version=ir_version), ie_device,
                    precision, temp_dir=temp_dir, ir_version=ir_version, use_old_api=use_old_api,
                    use_new_frontend=use_new_frontend, **params)
+
+
+class TestKerasMultipleOutputsWithActivation(CommonTF2LayerTest):
+    def create_keras_multiple_outputs_net(self, input_names, input_shapes, output_names, ir_version):
+        # create TensorFlow 2 model with multiple outputs
+        tf.keras.backend.clear_session()  # For easy reset of notebook state
+
+        inputs = []
+        outputs = []
+        for ind in range(len(input_names)):
+            input = tf.keras.Input(shape=input_shapes[ind][1:], name=input_names[ind])
+            inputs.append(input)
+            outputs.append(tf.keras.layers.Activation(tf.nn.sigmoid)(input))
+        tf2_net = tf.keras.Model(inputs=inputs, outputs=outputs)
+
+        # TODO: add reference IR net. Now it is omitted and tests only inference result that is more important
+        ref_net = None
+
+        return tf2_net, ref_net
+
+    test_data = [dict(input_names=["k", "b", "m", "c", "x"],
+                      input_shapes=[[1, 1], [1, 3], [1, 2], [1, 5], [1, 4]],
+                      output_names=["Func/PartitionedCall/output/_5:0",
+                                    "Func/PartitionedCall/output/_6:0",
+                                    "Func/PartitionedCall/output/_7:0",
+                                    "Func/PartitionedCall/output/_8:0",
+                                    "Func/PartitionedCall/output/_9:0"])]
```

Comment:
The problem of this test is that you don't specify names for output tensor. So user will work only with output orders and we should provide the same experience, i.e. to provide the same output order. The output tensor names theoretically can be changed from TF version to another. Now you check internal tensor names that can be changed in other TF version.

Here we only check the right output tensor (the right value after inference) by index using `get_output_tensor`: https://docs.openvino.ai/2023.1/api/ie_python_api/_autosummary/openvino.runtime.InferRequest.html

Additionally, you can add one more test where we can check output tensor names, in this case you need to pass dictionary of outputs for creation of tf.keras.Model:

```
outputs = {}
        for ind in range(len(input_names)):
            input = tf.keras.Input(shape=input_shapes[ind][1:], name=input_names[ind])
            inputs.append(input)
            outputs["name" + str(ind)] = tf.keras.layers.Activation(tf.nn.sigmoid)(input)
tf2_net = tf.keras.Model(inputs=inputs, outputs=outputs)
            outputs.append(tf.keras.layers.Activation(tf.nn.sigmoid)(input))

```

Reply:
Done.